### ISSUE TITLE
Pretrained checkpoints appear to be corrupted — unable to load for reproduction

### ISSUE THREAD

**Issue by pumblus** [HiLab-git/SicTTA#4]
Dear authors,

Thanks for releasing the code and checkpoints for SicTTA!

We are currently trying to reproduce the results from the paper. We trained our own source model using `train_source.py --cfg cfgs/mms/source.yaml`, but the Dice scores we get after adaptation are noticeably lower than the numbers reported in the paper. We were hoping to use your published checkpoint as a reference baseline to narrow down where the gap comes from.

Unfortunately, the checkpoint at `save_model/mms2d_unet/source-A-source-model-latest.pth` appears to be a corrupted file — torch.load fails with failed finding central directory regardless of PyTorch version. Looking at the raw bytes, the zip header (PK) is present but the payload is filled with 0x5A (Z) padding instead of valid serialized weights, which suggests the file may have been partially written or not fully uploaded:

```
$ xxd save_model/mms2d_unet/source-A-source-model-latest.pth | head -5
00000000: 504b 0304 0000 0808 0000 0000 0000 0000  PK..............
00000030: 0e00 5a5a 5a5a 5a5a 5a5a 5a5a 5a5a 5a5a  ..ZZZZZZZZZZZZZZ
```

This file was introduced in commit d28ad04 and also blocks any downstream command that references it (e.g. `python test_time_adaptation.py --cfg cfgs/mms/norm.yaml`). We also checked `save_model/Fundus-doFE_unet/source-Domain1-source-model-latest.pth` — same issue.

To help us get as close to the paper's reported performance as possible, would it be possible for you to re-upload a valid copy of the pretrained weights (for both M&MS-2D and Fundus if available)? A Google Drive or Zenodo link would work perfectly.

Thanks a lot for your help, and thanks again for making the code public!

**Comment by yjq-yes** [comment]
We met the same problem, and the `.pth` files also seem unusually small, so we suspect they may be incomplete, corrupted, or not the actual checkpoint files.

**Comment by JianghaoWu** [comment]
Thanks a lot for reporting this issue and for providing such a detailed diagnosis.

I have re-uploaded the pretrained checkpoints to Google Drive:
https://drive.google.com/drive/folders/12VgnWy8DOgrhtiJx_yfxP9bhqxIJPljJ?usp=drive_link

It seems that the previously released `.pth` files in the repository may have been corrupted or incompletely uploaded. Please use the checkpoints from the Google Drive link instead.

Thanks again for your interest in our work and for helping us identify this problem. Please feel free to let us know if there are still any issues with loading the checkpoints or reproducing the results.

**Comment by kingmzy** [comment]
Hi, thank you for releasing the code. I have a question regarding the Source-only baseline reproduction.

Using the released code, I obtained 76.60% for Source-only and 77.37% for SicTTA, while the paper reports 69.73% and 77.88%.

I noticed that the released code does not exactly match the 80/20 train-validation protocol described in the paper. After reproducing the patient-level 80/20 split and selecting the validation-best checkpoint, I obtained 75.87% for Source-only and 79.31% for SicTTA. This suggests that the split difference only explains about 0.73% of the baseline gap.

Could you please share any additional details about the Source-only training setup? For example, were there any differences in preprocessing, data filtering, optimization settings, checkpoint selection, or other implementation details that could explain why the reported Source-only baseline is 69.73%?

Thank you very much for your help.
